arithmetization/src/main/java/net/consensys/linea/zktracer/module/add/AddOperation.java

```diff
@@ -135,6 +132,7 @@ void trace(int stamp, Trace trace) {
 
   @Override
   protected int computeLineCount() {
-    return this.ctMax + 1;
+    ctMax = computeCtMax();
```

Comment:
if `res` is `final` why doing computations all the time when `computeLineCount` is called?

Reply:
I've seen same case for other `ModuleOperations` too

Reply:
you mean that we could compute res only when needed ? Yes that's true, modified